README.md

```diff
@@ -1,5 +1,61 @@
 # Filebrowser action for neuro-flow
 
-This is a [`neuro-flow`](https://github.com/neuro-inc/neuro-flow) action launching [Filebrowser](https://hub.docker.com/r/filebrowser/filebrowser) app for working with platform storage.
+This is a [`neuro-flow`](https://github.com/neuro-inc/neuro-flow) action launching the [Filebrowser](https://hub.docker.com/r/filebrowser/filebrowser) app for working with platform storage.
 
-All it needs is a reference to a storage folder that becomes root folder for filebrowser instance.
\ No newline at end of file
+All it needs is a reference to a storage folder that becomes the root folder for the Filebrowser instance.
+
+### Quick example
+
+```
+jobs:
+  filebrowser:
+    action: gh:neuro-actions/filebrowser@master
+    args:
+      volumes_project_remote: $[[ volumes.project.remote ]]
+```
+
+## Arguments
+
+### `http_port`
+
+HTTP port to use for Jupyter. `"80"` by default.
```

Comment:
Could you please replace jupyter -> filebrowser everywhere in this repo?
Our fault 😅 🙈